scripts/west/zap_generate.py

```diff
@@ -81,3 +85,14 @@ def do_run(self, args, unknown_args):
             self.check_call(self.build_command(zap_file_path, output_path, templates_path))
 
         log.inf(f"Done. Files generated in {output_path}")
+
+    def clear_generated_files(self, path: Path):
+        for file in path.iterdir():
+            if file.is_file():
+                with open(file, 'r') as f:
+                    for line in f.readlines():
+                        if "// THIS FILE IS GENERATED BY ZAP" in line:
+                            print(f"Removing {file}")
```

Comment:
```suggestion
                            log.dbg(f"Removing {file}")
```

To support west logging levels.